src/k8s-configuration/azext_k8s_configuration/custom.py

```diff
@@ -53,7 +53,7 @@ def show_k8s_configuration(client, resource_group_name, cluster_name, name, clus
 # pylint: disable=too-many-locals
```

Comment:
Update the help as well (../src/k8s-configuration/azext_k8s_configuration/_help.py)

Reply:
Good catch 😃